docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  speech-container:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: speech-container
+    working_dir: /app
+    volumes:
+      - .:/app
+    networks:
+      - speech-network
+    command: ["python", "server.py", "--port", "2224", "--whispermodel", "tiny"]
+  caddy:
+    image: caddy:latest
+    volumes:
+      - ./Caddyfile:/etc/caddy/Caddyfile
```

Comment:
Best to avoid unless a value in the volume; brittle, someone will delete the file, etc.  We should just do a Dockerfile and build it too since we are building and copy it in.  again, could have this a built image all ready to go for our users. caddy-gen could be interesting but a build off this caddy image and copying in the ccaddyfile will do the job for now.

Reply:
https://github.com/ClinicianFOCUS/speech2text-container/pull/4/commits/a2de59235e218e2573572959f8438cda0c19fb7c